scripts/setup-centos8.sh

```diff
@@ -86,9 +91,47 @@ wait  # For cmake and source downloads to complete.
   ./b2 "-j$(nproc)" -d0 install threading=multi
 )
 
-cmake_install_deps gflags -DBUILD_SHARED_LIBS=ON -DBUILD_STATIC_LIBS=ON -DBUILD_gflags_LIB=ON -DLIB_SUFFIX=64 -DCMAKE_INSTALL_PREFIX:PATH=/usr
-cmake_install_deps glog -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX:PATH=/usr
-cmake_install_deps snappy -DSNAPPY_BUILD_TESTS=OFF
-cmake_install_deps fmt -DFMT_TEST=OFF
+(
+  cd protobuf
+  ./configure --prefix=/usr
+  make "-j${NPROC}"
+  make install
+  ldconfig
+)
+
+cmake_install gflags -DBUILD_SHARED_LIBS=ON -DBUILD_STATIC_LIBS=ON -DBUILD_gflags_LIB=ON -DLIB_SUFFIX=64 -DCMAKE_INSTALL_PREFIX:PATH=/usr
+cmake_install glog -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX:PATH=/usr
+cmake_install snappy -DSNAPPY_BUILD_TESTS=OFF
+cmake_install fmt -DFMT_TEST=OFF
+cmake_install folly -DFOLLY_HAVE_INT128_T=ON
+# remove in #7700
+sed -i 's/\[\[deprecated.*\]\]//g' /usr/local/include/folly/init/Init.h
```

Comment:
Can you add some comment on why we are doing this ?

Reply:
This will go away once we upgrade folly. This was added when we upgraded fmt and it is temporary.